### PR TITLE
Pushes version to 1.15

### PR DESCRIPTION
**Pull request by DeJayDev**
It was already 1.15, just didn't have the right text in readme

### CODE REVIEW

File: src/main/java/net/starlegacy/explosionregen/package-info.java [hunk 1]
```diff
@@ -1,4 +0,0 @@
-@ParametersAreNonnullByDefault
```
Comment (MicleBrick) [inline src/main/java/net/starlegacy/explosionregen/package-info.java:1]: Why is this removed?

File: .idea/misc.xml [hunk 2]
```diff
@@ -6,6 +6,9 @@
     </list>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="JavaScriptSettings">
```
Comment (MicleBrick) [inline .idea/misc.xml:9]: What's up with this?

File: build.gradle [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apply plugin: "java"
 apply plugin: "maven"
 
-version = "2.5.0"
+version = "2.6.0"
```
Comment (MicleBrick) [inline build.gradle:4]: Given that there's no feature changes I don't think this warrants a version change. Usually version changes are only made when I make a release

File: README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ The only command is "/regen" to regenerate all pending blocks, requires the expl
 # Contributing
 Fork the project.
 Run these commands (use git bash if you're on Windows) (Requires Maven to be properly installed)
-```Bash
+```bash
```
Comment (MicleBrick) [inline README.md:14]: Does it make a difference?

File: src/main/java/net/starlegacy/explosionregen/package-info.java (hunk 1 above)
Comment (DeJayDev) [reply src/main/java/net/starlegacy/explosionregen/package-info.java:1]: What is it doing that helps

File: .idea/misc.xml (hunk 2 above)
Comment (DeJayDev) [reply .idea/misc.xml:9]: IntellIJ is weird, strange side effect of having your .idea folder on VCS (Version Control)

**Comment by MicleBrick** [comment]
I'd appreciate it if you make separate pull requests for each change

File: build.gradle (hunk 3 above)
Comment (DeJayDev) [reply build.gradle:4]: Version change was made with the intention I'd be updating the plugin to 1.15 since the readme was wrong.

File: src/main/java/net/starlegacy/explosionregen/package-info.java (hunk 1 above)
Comment (MicleBrick) [reply src/main/java/net/starlegacy/explosionregen/package-info.java:1]: It makes it so that parameters are non null by default.

File: README.md (hunk 4 above)
Comment (DeJayDev) [reply README.md:14]: Semantics, I was in the file fixing other things and didn't like it.

File: build.gradle (hunk 3 above)
Comment (MicleBrick) [reply build.gradle:4]: In that case, you should probably revert it

**Comment by DeJayDev** [comment]
This commit would be fine with only the README.md file merged, even if it's by hand now that I've read the review.

File: README.md (hunk 4 above)
Comment (MicleBrick) [reply README.md:14]: I prefer it this way, so please leave it

Comment (rsammelson) [reply README.md:14]: Bash actually isn’t recognized (https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). It should be `Shell` or `shell`.
